FirebaseStorage/Sources/Internal/StorageTokenAuthorizer.swift

```diff
@@ -0,0 +1,126 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+import FirebaseAppCheckInterop
+import FirebaseAuthInterop
+import FirebaseCore
+@_implementationOnly import FirebaseCoreExtension
+
+#if COCOAPODS
+  import GTMSessionFetcher
+#else
+  import GTMSessionFetcherCore
+#endif
+//
+//
+internal class StorageTokenAuthorizer: NSObject, GTMSessionFetcherAuthorizer {
+  func authorizeRequest(_ request: NSMutableURLRequest?,
```

Comment:
Should request be non-optional in this method (here and in other methods)?

Reply:
It needs to match the protocol.